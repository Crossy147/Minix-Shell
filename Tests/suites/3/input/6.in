# pipes - counting used descriptors

ls servers/pm > test.e4.3.A

cat test.e4.3.A test.e4.3.A test.e4.3.A | sort | wc

echo ------------------------
cat test.e4.3.A | sort | wc
echo ------------------------
cat test.e4.3.A | cat | cat | sort | cat | cat | cat | cat | cat | cat > test.e4.3.B

cat  test.e4.3.B | cat | cat |cat | cat | cat |cat | cat | cat |cat > test.e4.3.C
cat test.e4.3.C | cat | wc

rm test.e4.3.A test.e4.3.B test.e4.3.C

bin/fdcounter

bin/fdcounter > fds.1 | bin/fdcounter | bin/fdcounter | bin/fdcounter > fds.2 | bin/fdcounter | bin/fdcounter | bin/fdcounter | bin/fdcounter > fds.3 | bin/fdcounter | bin/fdcounter | bin/fdcounter 
 
cat fds.1 
cat fds.2
cat fds.3

rm fds.1 fds.2 fds.3
