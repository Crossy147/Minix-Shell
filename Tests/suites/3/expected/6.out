     63     63    501 
------------------------
     21     21    167 
------------------------
     21     21    167 
3 file descriptors used.
3 file descriptors used.
3 file descriptors used.
3 file descriptors used.
3 file descriptors used.
